src/modules/loadDataFromApi.js

```diff
@@ -0,0 +1,12 @@
+import renderOnPage from './renderOnPage.js';
+
+const loadDataFromAPI = async () => {
+  const baseApiUrl = 'https://www.themealdb.com/api/json/v1/1/filter.php?a=Canadian';
```

Comment:
kindly baseapiurl is a constant variable make it outside any block and capitalize it.